### DISCUSSION TITLE
What is the difference between Raw and RawQuery?

### DISCUSSION THREAD

**Discussion by RangelReale** [stephenafamo/bob#62] [General]
I can use both in `sm.Where` for example, but `Raw` surrounds the code in parenthesis, while `RawQuery` does not.

**Comment by stephenafamo** [comment]
`RawQuery` returns a `Query`, which has some extra methods like `Build`. However, since queries are also expressions (so that they can be used in places that take a subquery), it also works in many other places.

The parentheses are because whenever a Sub Query is used inside another query, it usually has to be surrounded by parentheses.
